packages/styles/dialog.css

```
@@ -100,3 +100,41 @@
   border-top: 1px solid var(--dialog-border-color);
   padding: calc(var(--space-small) - 4px) var(--dialog-padding);
 }
+
+/**
+* style for plain dialog
+*/
+
+.Modal--info .Dialog__header {
+  display: flex;
+  justify-content: center;
+  border: none;
+  background-color: white;
+  height: auto;
+}
+
+.Modal--info .Dialog__heading {
+  padding-top: 24px;
+  height: auto;
+  margin-right: 16px;
+  text-align: center;
+}
```

```suggestion
.Modal--info .Dialog__heading {
  flex: 1;
  justify-content: center;
  padding-top: var(--space-large);
  height: auto;
  margin-right: var(--space-small);
}
```

This is based off changes to `.Dialog__header`:

* Uses appropriate css variables
* Center aligns the text using flex values

---

I am keeping text-align, otherwise if I pass in a <h1>, it will be left-aligned. Or is there a better way?

---

<h1> This is fun!

---

hmm, now I think about it. I guess we can leave that out of Cauldron